templates/taxi/driver_detail.html

```diff
@@ -1,15 +1,16 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    Username: {{ driver.username }}
-  </h1>
+  <h1>Username: {{ driver.username }}</h1>
 
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
   <p><strong>License number:</strong> {{ driver.license_number }}</p>
   <p><strong>Is staff:</strong> {{ driver.is_staff }}</p>
 
+  <a href="{% url 'taxi:driver-license-update' pk=driver.id %}" class="btn btn-secondary">Update License</a>
```

Comment:
Consider adding a 'Cancel' button that leads to the previous page the user was on, as per the checklist requirement.